cli/parse_htmls.py

```diff
@@ -6,11 +6,11 @@
 
 import cloudpathlib.exceptions
 from cloudpathlib import CloudPath
+from cpr_data_access.parser_models import ParserInput, ParserOutput, HTMLData
```

Comment:
i'm not sure which PR the pyproject.toml update is to install `data-access` but just wanted to make sure it's set to a fixed version

Reply:
It's installed as a dependency of the azure pdf parser so yes should be fixed as it's fixed within the azure pdf parser deps